openmrs/src/test/java/org/motechproject/openmrs/resource/impl/ProgramEnrollmentResourceImplTest.java

```diff
@@ -96,6 +98,25 @@ public void shouldUpdateProgramEnrollment() throws Exception {
     }
 
     @Test
+    public void shouldUpdateBahmniProgramEnrollment() throws Exception {
+        ProgramEnrollment programEnrollment = prepareProgramEnrollmentWithAttributes();
+
+        URI url = config.toInstancePathWithParams("/bahmniprogramenrollment/{uuid}", programEnrollment.getUuid());
+
+        when(restOperations.exchange(eq(url), eq(HttpMethod.POST), any(HttpEntity.class), eq(String.class)))
+                .thenReturn(getResponseFromFile(PROGRAM_ENROLLMENT_RESPONSE));
+
+        ProgramEnrollment updated = programEnrollmentResource.updateBahmniProgramEnrollment(config, programEnrollment);
+
+        verify(restOperations).exchange(eq(url), eq(HttpMethod.POST), requestCaptor.capture(), eq(String.class));
+
+        assertThat(updated, equalTo(programEnrollment));
```

Comment:
ProgramEnrollment does not have attributes in equals method, so this check does not verify attributes